src/main/java/org/spongepowered/api/item/recipe/Recipe.java

```diff
@@ -48,7 +47,7 @@
      *
      * @return the results of this {@link Recipe}
      */
-    Collection<ItemStackSnapshot> getResults();
+    ImmutableCollection<ItemStackSnapshot> getResults();
```

Comment:
I would prefer if this returns any kind of `Collection` instead of this specialized one.